### PR TITLE
Sync the tika-guide with the apache-tika quickstart

### PR DESCRIPTION
**Pull request by sberyozkin**

### CODE REVIEW

**Comment by sberyozkin** [comment]
Hi @cescoffier and @gsmet if there is still time for this update then please review/approve, would not be a problem I guess if it will make it a bit later for 0.26.0, thanks

File: docs/src/main/asciidoc/tika-guide.adoc [hunk 1]
```diff
@@ -126,3 +94,141 @@ public class TikaParserResource {
 }
 ----
 
+== Creating the Maven Project
```
Comment (gsmet) [inline docs/src/main/asciidoc/tika-guide.adoc]: Could you try to have your doc structured as the others? Typically https://github.com/quarkusio/quarkus/blob/master/docs/src/main/asciidoc/hibernate-search-guide.adoc#solution with the Solution section then the project creation.

File: docs/src/main/asciidoc/tika-guide.adoc [hunk 2]
```diff
@@ -126,3 +94,141 @@ public class TikaParserResource {
...
+    <dependency>
+        <groupId>io.quarkus</groupId>
+        <artifactId>quarkus-resteasy</artifactId>
+    </dependency>
+----
+
+
+=== Examine the JAX-RS resource
+
+Open the `src/main/java/org/acme/quickstart/tika/TikaParserResource.java` file and see the following content:
```
Comment (gsmet) [inline docs/src/main/asciidoc/tika-guide.adoc]: `see`? `paste` maybe? Because the just created file will not contain this.

**Comment by sberyozkin** [comment]
Added to a 0.25.0 milestone, np if you decide to move to 0.26.0 :-)

File: docs/src/main/asciidoc/tika-guide.adoc [hunk 3]
```diff
@@ -126,3 +94,141 @@ public class TikaParserResource {
...
+[source,shell]
+----
+$ curl -X POST -H "Content-type: Content-type: application/vnd.oasis.opendocument.text" --data-binary @target/classes/quarkus.odt http://localhost:8080/parse/text
+Hello Quarkus
+----
+
+== Explore the Solution
+
+The solution repository located in the `apache-tika` {quickstarts-archive-url}[directory] contains the examples we have worked through in this quickstart guide.
+We suggest that you check out the quickstart solutions and explore the `apache-tika` directory to learn more about the {extension-name} extension features.
```
Comment (gsmet) [inline docs/src/main/asciidoc/tika-guide.adoc]: And you should get rid of that part once you have a structure similar to the other guides.

**Review by gsmet — CHANGES_REQUESTED** [review]
I added a few comments and suggestions.

**Comment by sberyozkin** [comment]
@gsmet Have a look please; I'll have to see GP soon so will be offline shortly, thanks for the review so far; np if the doc will need a bit more work to do and won't make it :-)

**Review by gsmet — APPROVED** [review]
I think it's good enough for this release, we can improve later.

**Comment by sberyozkin** [comment]
@gsmet definitely :-) thanks

**Pull request merged**